.github/workflows/code-freeze.yml

```diff
@@ -0,0 +1,153 @@
+# This is a basic workflow to help you get started with Actions
+
+name: Code Freeze Bot
+on: workflow_call
+
+env:
+  PMC_MEMBERS: '[ "gdams", "sxa", "johnoliver", "tellison", "jerboaa", "smlambert", "karianna", "llxia", "sanhong" ]'
+
+permissions:
+  contents: write
+  pull-requests: write
+
+jobs:
+  freeze:
+    runs-on: ubuntu-latest
+    # Change to false when code freeze is not in place
+    if: true
```

Comment:
Works for now but would prefer this checkout and test for existence/content of a file in the .github repo to determine global freeze status